projects/igniteui-angular/src/lib/query-builder/query-builder.component.html

```diff
@@ -86,9 +81,6 @@ <h6 class="igx-filter-empty__title">
         >
             <igx-chip
                 [data]="expressionItem"
-                [displayDensity]="
-                    displayDensity === 'compact' ? 'cosy' : displayDensity
```

Comment:
All QB removals are based on the fact that @simeonoff's team would reimplement the same behavior within the themes. This is not done yet, though I did test and QB looked quite alright without the display densities, so it might have been already addressed. I'll test it a bit further.

Reply:
I dont think that it looks good. These sizes are set for a reason. If I remember correct, Simeon told me that some of those come from Jason. There are couple of these in the grid as well. They are limited to a specific size because visually they do not look good if for example the select component is set other than compact.